src/main/resources/jenkins/plugins/http_request/HttpRequest/config.jelly

```diff
@@ -37,21 +38,7 @@
             <f:select />
         </f:entry>
         <f:entry title="Custom headers">
-            <f:repeatable field="customHeaders">
-                <table width="100%">
-                    <f:entry title="Header" field="name">
-                        <f:textbox/>
-                    </f:entry>
-                    <f:entry title="Value" field="value">
-                        <f:textbox/>
-                    </f:entry>
-                    <f:entry>
-                        <div align="right">
-                            <f:repeatableDeleteButton/>
-                        </div>
-                    </f:entry>
-                </table>
-            </f:repeatable>
+            <f:repeatableProperty field="customHeaders" />
```

Comment:
Hmm, should work (since https://github.com/jenkinsci/jenkins/commit/96187b71e7188fcc137a450001328fb4d99c2c48), not sure what is wrong in this case.